projects/wipp-frontend-lib/README.md

```diff
@@ -0,0 +1,24 @@
+# WippFrontendLib
+
```

Comment:
Can you update this readme to be more reflective of what the lib folder contains? Maybe a table listing all the different components? Instructions on how to use those components in an application